commander/src/utils/reader.ts

```diff
@@ -352,3 +357,27 @@ export const getParamsFromPrompt = async (
 		? transformNestedAsset(assetSchema, filteredResult)
 		: transformAsset(assetSchema, result as Record<string, string>);
 };
+
+export const getFileExtension = (filePath: string): string => {
+	const ext = path.extname(filePath);
+
+	if (!ext || ext !== '.json') {
+		throw new ValidationError(INVALID_JSON_FILE);
+	}
+
+	return ext;
+};
+
+export const readParamsFile = (filePath: string): string => {
+	try {
+		const params = fs.readFileSync(filePath, 'utf8');
+		return params;
+	} catch (err) {
+		throw new ValidationError(FILE_NOT_FOUND);
+	}
+};
+
+export const getFileParams = (filePath: string): string => {
+	getFileExtension(filePath);
```

Comment:
Minor point: since there is no assignment of the returned value the function & name could be changed to only check and throw as intended.

Reply:
like instead of `getFileExtension` we have `checkFileExtension`?

Reply:
`checkFileExtension` sounds good

Reply:
Updated!